docs/docs/quick-start.md

```diff
@@ -1,81 +1,74 @@
----
-sidebar_position: 2
----
-
 # Quick Start
```

Comment:
This is a great foundation overall. I think you should add high-level info about the following:
- The Redwood CLI: help and link to doc
- Tests: CLI, files, and Doc
- Storybook: CLI, files, and Doc
- Deploy: both serverless and Fastify server, CLI setup and Doc
- Styling: Tailwind and Chakra, CLI setup

Those don't need a deep-dive, but mentioning them also communicates end-to-end, dev-to-deploy integrations